### PR TITLE
9 plugins manifests

### PR DESCRIPTION
**Pull request by zuckung**

### CODE REVIEW

**Comment by zuckung** [comment]
There is a forbidden url error. Link works. Just a tempoary github problem?

Checking manifests/navy.licenses.yaml
  ERROR 403	https://github.com/zuckung/endless-sky-plugins/releases/download/v1.0-navy.licenses/navy.licenses.zip

**Review by MCOfficer — COMMENTED** [review]
oh bother. this is a problem.

So you know how the autoupdate regularly checks your repo for new versions? it does that by taking all releases/tags and ordering them in a way that doesn't really matter here - the point is, it will decide that one of those tags must be the latest.

And it will decide on that one tag for *all of these plugins*.
We were lucky that `v1.1-galactic.captical.investment` actually *is* considered the latest out of all of these, so autoupdate hasn't tried to update that plugin to, say, `v1.0-better.starts`.

Bottom line is, autoupdate really isn't made for multirepos. It can work if you only add one plugin to the index, but any more and things will break.

**Comment by zuckung** [comment]
Ok that's a problem.  Please remove galactic capital again, cause I do a lot updates :/ 

In case you rework the script someday to be more tolerant, please tell me.
